core/src/main/java/hudson/XmlFile.java

```diff
@@ -126,8 +129,18 @@ public XmlFile(File file) {
     }
 
     public XmlFile(XStream xs, File file) {
+        this(xs, file, true);
+    }
+
+    /**
+     * @param force Whether or not to flush the page cache to the storage device with {@link
+     *     FileChannel#force} (i.e., {@code fsync}} or {@code FlushFileBuffers}) before this method
+     *     returns. If you set this to {@code false}, you will lose data integrity.
```

Comment:
Lacks `@since`

Reply:
I've addressed your comments in the past and still not gotten approvals from you, so I'm less inclined to address comments like this. Feel free to open a PR if you think this should be fixed.

Reply:
It's far easier with limited time to point out weaknesses than approve a nontrivial PR in its entirety.

This one isn't even personal preference (where your response would make sense), if there was a tool for this we'd use it.

Reply:
Perhaps a bit too easy, especially compared to writing the PR in the first place. By reviewing a PR, you are taking at least partial responsibility for shepherding the PR to completion. While all of us are guilty of so-called "drive-by reviews" from time to time, I think it is a behavior best avoided in healthy code review cultures.

We're getting off topic here. I opened #5661 for the `@since` annotation. Approve it — or not. It's up to you.

Reply:
Good point; I'll try to do better.